index.js

```diff
@@ -96,7 +96,7 @@ class NpmAddDependencies {
         return res;
       }, {});
 
-      json[this.getTargetName()] = this.result;
+      json[this.getTargetName()] = Object.assign(json[this.getTargetName()]||{}, this.result);
```

Comment:
What about doing that:
```
this.result = Object.assign(json[this.getTargetName()] || {}, this.result);
```

before sorting dependencies? Smth like:
```
this.result = Object.assign(json[this.getTargetName()] || {}, this.result);

this.result = Object.keys(this.result).sort().reduce((res, key) => {
    res[key] = this.result[key];

    return res;
}, {});

json[this.getTargetName()] = this.result;
```

So, that all dependencies in the list were sorted by alphabet.